cpp/cmake/thirdparty/get_cpm.cmake

```diff
@@ -0,0 +1,47 @@
+set(CPM_DOWNLOAD_VERSION 4fad2eac0a3741df3d9c44b791f9163b74aa7b07) # 0.32.0
```

Comment:
We should update to `0.32.1` which fixes a regression that has bitten some rapids users ( https://github.com/cpm-cmake/CPM.cmake/releases/tag/v0.32.1 ).